pom.xml

```diff
@@ -2280,26 +2292,6 @@
             <artifactId>lettuce-core</artifactId>
             <version>${lettuce.version}</version>
             <exclusions>
-            	<exclusion>
-            		<groupId>io.netty</groupId>
```

Comment:
Since we're currently keeping our POM entry for the Lettuce import, should we keep these Netty-related exclusions in place instead of removing them? Or are the other dependencies now using the same Netty version that this Lettuce import uses?

Reply:
Good point, adding back

Reply:
If I add the exclusions back in I get this error on startup:
`Caused by: io.netty.handler.codec.EncoderException: java.lang.NoClassDefFoundError: io/netty/util/internal/ObjectPool$ObjectCreator
	at io.netty.handler.codec.MessageToByteEncoder.write(MessageToByteEncoder.java:125)`

Reply:
What does the POM dependency hierarchy look like for the Netty-related JARs, when you click the Dependency Hierarchy tab on an opened POM in Eclipse?

Based on what I'm seeing in the develop branch, I think the exclusions were initially added so that Lettuce's Netty JARs wouldn't interfere with the ones brought in by the "secretsmanager" dependency. If "secretsmanager" is no longer bringing in those JARs (or suitable versions of those JARs), then further adjustments to the JAR exclusions may be needed.

Reply:
This has been updated